packages/cypress/src/migrations/update-9-4-0/add-cypress-eslint-plugin-9-4-0.ts

```diff
@@ -6,6 +6,7 @@ import {
   updateJsonInTree,
   readWorkspace,
 } from '@nrwl/workspace';
+import { noop } from 'rxjs';
```

Comment:
Should this come from `@angular-devkit/schematics`?

Reply:
Somehow.. this worked :thinking: lol... guess a noop is a noop no matter where it comes from xD